expressions.js

```diff
@@ -119,7 +118,7 @@ module.exports = {
       const redirectToAfterLogout = validRedirectUrl(redirectUri)
         ? redirectUri
         : `https://${VEGA_URL}/search`
-      return `${CAS_SERVER_DOMAIN}/iii/cas/logout?service=${redirectToAfterLogout}`
+      return `${VEGA_AUTH_DOMAIN}/auth/realms/nypl/protocol/openid-connect/logout?redirect_uri=${redirectToAfterLogout}`
```

Comment:
where does the redirect_uri param get used?

Reply:
We pass a `?redirect_uri=` param to our Logout route any time we want to patron to return to somewhere specific at the end of logging out. For example, when logging out of RC on a specific bib detail page, ideally they return to that same page. If no URI is given, it defaults to Vega home.